packages/edit-site/src/components/sidebar-edit-mode/template-panel/index.js

```diff
@@ -7,7 +7,7 @@ import { store as editorStore } from '@wordpress/editor';
 import { store as coreStore } from '@wordpress/core-data';
 import { decodeEntities } from '@wordpress/html-entities';
 import { __ } from '@wordpress/i18n';
-import { navigation as navigationIcon } from '@wordpress/icons';
+import { navigation, symbol } from '@wordpress/icons';
```

Comment:
Do we always use the `Icon` suffix, or can we omit it when there's no name collision?